src/modules/workspaces/workspaces.usecase.ts

```diff
@@ -2668,4 +2670,24 @@ export class WorkspacesUsecases {
       }
     }
   }
+
+  async searchWorkspaceContent(
+    user: User,
+    workspaceId: Workspace['id'],
+    search: string,
+  ) {
+    const workspace = await this.workspaceRepository.findById(workspaceId);
```

Comment:
We would like this endpoint to be faster as possible. You can eliminate this if you want, the guard is already checking if the workspace exists, tbh the database shouldn't take too much time looking for this, but time is time :)